docs/axes/cartesian/index.md

```diff
@@ -283,8 +283,8 @@ module.exports = {
 };
 ```
 
-::: tip
-The `crossAlign` setting is not used the the tick rotation is not `0`, the axis position is `'center'` or the position is with respect to a data value.
+:::tip Note
+The `crossAlign` setting is not used the tick rotation is not `0`, the axis position is `'center'` or the position is with respect to a data value.
```

Comment:
I know this is what you were fixing, but lets fix it as it was noticed :)

I think the first `the` should be `when`, currently it does not make sense to me.
Even then the double negative makes it non-optimal.

How about something like:
```suggestion
The `crossAlign` setting is only effective when these preconditions are met:
  * tick rotation is `0`
  * axis is position is `'top'`, '`left'`, `'bottom'` or `'right'`
```

Lets wait what @etimberg has to say to this suggestion before updating :)

Reply:
Would remove the first `is` is the last line so it would become: `* axis position is ....`

Reply:
I like that @kurkle. @LeeLenaleee I do agree that the first `is` shouldn't be there